en/docs/apis/create-and-publish-your-first-api.md

```diff
@@ -2,38 +2,32 @@
 
 Choreo’s API management capabilities allow you to create, publish, and manage all aspects of an API and its lifecycle.
 This quick start guide walks you through the steps to quickly create and publish a REST API using Choreo. 
-
-## Prerequisites
-- A mock REST service to use when creating the API. 
-    
-    !!!note
-        In this quick start guide, let's use the mock service at [http://www.mocky.io/v2/5185415ba171ea3a00704eed](http://www.mocky.io/v2/5185415ba171ea3a00704eed), which is made available by [https://www.mocky.io/](https://www.mocky.io/). Click the mock service URL to test the service. You'll see the following JSON message: `{"hello": "world"}`.
  
 ## Step 1: Create an API from scratch
-Follow this procedure to create an API using the mock REST service you tested in the prerequisites section:
+Follow this procedure to create an API:
 
 1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
 2. Go to the **APIs** card and click **Get Started**. This takes you to the **Create API** page. 
     
     !!!note
-        If this is not the first API you are creating, you must click **+ Create** to go to the **Create API** page.
+        If this is not the first API you are creating,  click **+ Create** to go to the **Create API** page.
 
 3. On the **Create API** page, go to the **Start from scratch card** and click **Next**.
 4. In the **Create API from REST API** form, enter the details as follows:
     
     | **Field**        | **Value**                                        |
     |------------------|--------------------------------------------------|
-    | **API Name**     | `HelloWorld`                                     |
+    | **API Name**     | `covid19`                                     |
     | **API Version**  | `1.0.0`                                          |
-    | **Endpoint**     | `http://www.mocky.io/v2/5185415ba171ea3a00704eed`|
+    | **Endpoint**     | `https://disease.sh`|
 
 5. Click **Create**. This displays an overview of the API.
 
     ![API overview](../assets/img/apis/api-overview.png){.cInlineImage-full}
        
 6. Under **API Configuration**, click **Resources**. Then follow this procedure to add a resource to the API:
     1. Select `GET` as the HTTP verb.
```

Comment:
Shouldn't we bold `HTTP verb` as we are referring to UI field name here.

Reply:
done